main/windows/frames/index.ts

```diff
@@ -125,6 +127,12 @@ export default class FrameManager {
   broadcast (channel: string, args: any[]) {
     Object.keys(this.frameInstances).forEach(id => this.sendMessageToFrame(id, channel, ...args))
   }
+
+  reloadFrames () {
```

Comment:
what's this for, is it only used for development?

Reply:
yeah, for now anyway